ruby/import_js/importer.rb

```diff
@@ -164,11 +164,18 @@ def find_files(variable_name)
       if alias_path = @config.get('aliases')[variable_name]
         return [alias_path]
       end
+      dashed_variable = camelcase_to_dashed(variable_name)
       snake_case_variable = camelcase_to_snakecase(variable_name)
       matched_files = []
       @config.get('lookup_paths').each do |lookup_path|
         Dir.chdir(lookup_path) do
-          matched_files.concat(Dir.glob("**/#{snake_case_variable}.js*"))
+          matched_files.concat(
+            Dir.glob([
+              "**/#{dashed_variable}.js*",
+              "**/#{snake_case_variable}.js*",
+              "**/#{variable_name}.js*"
+            ])
```

Comment:
I'm about to push a new version - the '**/icon*button.js*' has the best perf., still need to do some QA to see how well it works.